app/controllers/api/v1/recipe_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::RecipeController < ApplicationController
```

Comment:
Where did this file come from?

Reply:
@vhoof I'm not sure how this made it into our PR. Neither Fransisco or I have made changes to this file. All i can think of is that it got pushed because I merged staging into our branch.

Reply:
We should probably remove it. The actual file is `recipes_controller`, so not sure what this one is. And either way, it shouldn't be in your PR.